docs/index.md

```diff
@@ -52,16 +52,21 @@ You would need to install an ESLint plugin for your editor first.
 Ever since ESLint 6, global installs of ESLint configs are no longer supported.
 To work around this new limitation, while still supporting in-editor linting, we've created a new [eslint-config-enact-proxy](https://github.com/enactjs/eslint-config-enact-proxy) package.
 The [eslint-config-enact-proxy](https://github.com/enactjs/eslint-config-enact-proxy) acts like a small proxy config, redirecting ESLint to use a globally-installed Enact ESLint config.
+
+```sh
+npm install eslint-config-enact-proxy
```

Comment:
Why did you put this here?

Reply:
I will revert this change

Reply:
I think some developers may use this example as a usage guide of `eslint-config-enact-proxy`.
Installing `eslint-config-enact-proxy` is also an important step but there's no guidance for installing it.